versioned_docs/version-3.0.0-beta/framework-guide/deploy-angular.md

```diff
@@ -0,0 +1,66 @@
+---
+id: deploy-angular
+title: How to Deploy Angular Apps in less than 30 Seconds
+sidebar_label: Deploy Angular App
+slug: /framework-guide/deploy-angular
+---
+
+This guide will show you how to deploy a Angular project and set up your domain.
+
+[Angular](https://angular.io/) is a popular JavaScript framework for building apps across all platforms; web, mobile web, native mobile, and native desktop.
+
+## Deploy Angular to Spheron
+
+Follow these steps to deploy your first Angular app on Spheron successfully.
+
+### Step 1: Put Your Angular Project Code in a Git Repo
+
+While this feature is not new to developers, the magic of Spheron is that every push to any branch in your GitHub, GitLab, or Bitbucket repo triggers a new build and deployment of your project.
+
+### Step 2: Update Your Vue.js Project for Deploying on Decentralised Storage
```

Comment:
### Step 2: Update your Angular Project for Deploying on Decentralised Storage

Reply:
thanks for pointing this out